hpOneView/connection.py

```diff
@@ -148,6 +149,9 @@ def do_http(self, method, path, body, custom_headers=None):
                     conn.close()
                 time.sleep(1)
                 continue
+            except http.client.HTTPException as e:
+                raise HPOneViewException(str(e))
```

Comment:
Perhaps its worth using `traceback.format_exc()` here instead of `str(e)`, in a similar way to [this line](https://github.com/HewlettPackard/python-hpOneView/blob/master/hpOneView/connection.py#L438), to make sure we get the same behavior in py2/3 and a full stack trace in case of hitting an issue.

Reply:
Done

Reply:
This line appears to still be using `str(e)` though, perhaps it was missed in the commit?

Reply:
Missed that one, fixed